pkg/apis/config.openshift.io/v1/register.go

```diff
@@ -36,7 +36,7 @@ func init() {
 // Adds the list of known types to api.Scheme.
 func addKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(SchemeGroupVersion,
-		&CVOConfig{},
+		&ClusterVersion{},
 		&CVOStatus{},
```

Comment:
this stands out now.

Reply:
Yeah, I think the spec/status split discussion can happen as a second step here.  I will create a PR that has those additional commits so we can look at it.